### ISSUE TITLE
Outdated summit script

### ISSUE THREAD

**Issue by kswirydo** [LLNL/hiop#524]
When I source scripts/summitVariables I am getting the following errors:
 source scripts/summitVariables.sh 
`Lmod has detected the following error:  The following module(s) are unknown: "gcc/7.4.0"`
```
Lmod has detected the following error:  These module(s) or extension(s) exist but cannot be loaded as requested: "m4/1.4.18"
   Try: "module spider m4/1.4.18" to see how to load the module(s).
Lmod has detected the following error:  The following module(s) are unknown: "python/3.7.0"
```

Lmod has detected the following error:  The following module(s) are unknown: "openmpi/4.0.3"

```
Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore_cache load "openmpi/4.0.3"
```

**Comment by pelesh** [comment]
Similar issue is addressed in #522 for newell.

**Comment by jaelynlitz** [comment]
I don't quite have access to Summit yet, maybe this could go to someone at ORNL in the meantime
